.github/workflows/integration.yml

```diff
@@ -0,0 +1,16 @@
+# This workflow runs a schedule and uses the latest version of tt-mlir (origin/main branch)
+# to check if the latest version of tt-mlir is compatible with the current codebase.
+
+name: Integration
```

Comment:
Do you plan to monitor these scheduled jobs and report us back when there is an issue? Do you have some workflow planed out for these crons?

Reply:
Yes, the workflow will be as stated in the ticket 
"If tests fail create a P0 issue and assign it to @nvukobratTT and @sdjordjevicTT", after that you can triage and see if it is mlir or forge issue, or both, and who will do the repair. Triage is P0, and you can see if there is a quick fix, workaround, or bigger repair item is needed.
We can automate ticket creation, but I'm afraid that it can start making duplicates if it's not resolved in a day, so we will start with manual monitoring of these jobs and see how it goes.

Reply:
@sdjordjevicTT what are your thoughts? Do you agree that these should be P0 as well?